src/delimiter.js

```diff
@@ -353,21 +351,24 @@ const sqrtSpan = function(height, delim, options) {
         sizeMultiplier = newOptions.sizeMultiplier / options.sizeMultiplier;
         spanHeight = 1 * sizeMultiplier;
         span = sqrtSvg("sqrtMain", spanHeight, viewBoxHeight, options);
+        span.style.minWidth = "0.781em";
         span.surdWidth = 0.833 * sizeMultiplier;   // from the font.
```

Comment:
Why is `minWidth` different from `surdWidth`?

Reply:
It's not the best choice of variable names. For the CSS `minWidth`, I used the actual width of the ink in the glyph. What I have called `surdwidth` is actually the horizontal advance value from the font.  Horizontal advance value is of course, a little wider than the ink.

I might adjust the `surdwidth` name and re-commit.

Reply:
I will also adjust the values of `min-width`. The paths in these surds have the `xMin` value baked into the SVG paths. So I'll increase each `min-width` by that amount.